docs/sphinx/using/backends/hardware.rst

```diff
@@ -312,6 +312,16 @@ configuration.
 
   export ORCA_ACCESS_URL="https://<ORCA API Server>"
 
+
+Sometimes the requests to the PT-1 require an authentication token. This token can be set as an
```

Comment:
I just want to confirm that the authentication token is only required 'sometimes' (not always).

Reply:
I can confirm that is the case